### DISCUSSION TITLE
How to replace a word in a file or peoject?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1073] [Q&A] [answered]
Personlly, I replace a word by `noremap <LEADER>sw :%s//g<left><left>` or by a plugin named `far.vim`. It's a common question for `IDE`.

**Comment by yinghaoyu** [comment]
Not lsp rename

**Reply by ayamir** [reply, marked as answer]
https://github.com/ayamir/nvimdots/blob/main/lua/keymap/completion.lua#L40-L44

**Reply by yinghaoyu** [reply]
Thanks.I see that.

**Comment by ayamir** [comment]
FYI: https://github.com/nvim-pack/nvim-spectre
